components/constellation/constellation.rs

```diff
@@ -976,6 +977,13 @@ impl<Message, LTF, STF> Constellation<Message, LTF, STF>
                 debug!("constellation got CloseBrowser message");
                 self.handle_close_top_level_browsing_context(top_level_browsing_context_id);
             }
+            // Panic a top level browsing context.
+            FromCompositorMsg::SendError(top_level_browsing_context_id, error) => {
+                debug!("constellation got SendError message");
```

Comment:
You want to add a `warn!()` here for when there is no browser.

Reply:
added it, a few lines below...

Reply:
Oh right.